nuxeo-core/nuxeo-core-bulk/src/main/resources/OSGI-INF/bulk-config.xml

```diff
@@ -4,25 +4,38 @@
   <require>org.nuxeo.runtime.stream.service</require>
 
   <extension target="org.nuxeo.ecm.core.bulk" point="actions">
-    <action name="setProperties"/>
+    <action name="setProperties" />
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.bulk" point="actions">
+    <action name="setSystemProperties" />
   </extension>
 
   <extension target="org.nuxeo.runtime.stream.service" point="streamProcessor">
+
     <!-- Processor used by BulkService to scroll DB -->
     <streamProcessor name="bulkDocumentScroller" class="org.nuxeo.ecm.core.bulk.StreamBulkProcessor"
-                     logConfig="bulk" defaultConcurrency="1" defaultPartitions="1">
+      logConfig="bulk" defaultConcurrency="1" defaultPartitions="1">
       <option name="scrollBatchSize">100</option>
       <option name="scrollKeepAlive">60</option>
       <option name="bucketSize">50</option>
       <option name="counterThresholdMs">500</option>
     </streamProcessor>
 
     <!-- SetProperty processor -->
-    <streamProcessor name="setProperties" class="org.nuxeo.ecm.core.bulk.actions.SetPropertiesAction" logConfig="bulk"
```

Comment:
Line shouldn't be changed, its length is 119.

Reply:
https://doc.nuxeo.com/corg/xml-or-xml-like-files-code-style/